ISISPostProcessRPM/rpmbuild/autoreduce-mq/usr/bin/QueueProcessor/settings.py

```diff
@@ -3,15 +3,15 @@
 FACILITY = 'ISIS'
 
 MYSQL = {
-    'HOST': 'reducedev2.isis.cclrc.ac.uk',
```

Comment:
Something to think about: Perhaps we should have a settings_dev.py also which we can either switch over to for dev environment - or some other option that install easier on dev environment